### PR TITLE
Fix a broken link in doc

### PR DESCRIPTION
**Pull request by yanana**
Fixed a broken link in routes.md.

### CODE REVIEW

**Comment by travisbrown** [comment]
Changing the brackets looks good to me, but I think there were two errors here—it says `router.md` instead of `route.md`.

File: docs/route.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ val users =
 ### Endpoints
 
 **Important:** endpoints are deprecated in 0.7.0 (and will be removed in 0.8.0) in favour of
-[Coproduct Routers][router.md#coproduct-routers].
+[Coproduct Routers](router.md#coproduct-routers).
```
Comment (vkostyukov) [inline docs/route.md]: As @travisbrown mentioned, it should be `route.md`. @yanana could you please update also fix it?

**Comment by yanana** [comment]
Thanks for pointing out. I fixed two errors.

**Comment by vkostyukov** [comment]
Nice! Thanks @yanana!

**Pull request merged**